source/_docs/zigbee.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: "Zigbee"
+description: "Using Zigbee with Home Assistant."
+redirect_from: /getting-started/zigbee/
```

Comment:
? Why this redirect?

Reply:
Miss from when first copied the main text from the Z-Wave documentation. Removed now.